service/ansible/roles/atmo-common/vars/main.yml

```diff
@@ -16,3 +14,5 @@ sshkeys:
   - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDFE/lyzLdFYZF3mxYwzrTITgov1NqtLuS5IY0fgjpdiVpHjWBUdXspTafKORbbM+t0ERTOqcSt24Vj5B8XUXImpzw2OAsl//AiKvHGRUenk7qY6/9IEUcay5mGAoiRpjLzDIDdtiQUAAEMKvkzanUBQOBJWVyO4Gq2aFUr4zweVLfvjejOspf2cZll/ojcPYmI9cKMq7fOgKSmRH2zUg+ORFlP1rQYugoETcGkcQg0IBsSMLT8gnYt3UWTW8S8ugtb4aaWVrId14Nc3sk+yDzPBaRX7iM3CQ5uKXPwjeID59RLMjQUFlHjqDSdZBOjXCFRHZbrbZZjS42o4OJAoLvF sgregory@mickey"
   - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQNBua13LVIG61LNztP9b7k7T+Qg8t22Drhpy17WVwbBH1CPYdn5NR2rXUmfiOa3RhC5Pz6uXsYUJ4xexOUJaFKY3S8h9VaeaPxMyeA8oj9ssZC6tNLqNxqGzKJbHfSzQXofKwBH87e+du34mzqzm2apOMT2JVzxWmTwrl3JWnd2HG0odeVKMNsXLuQFN6jzCeJdLxHpu+dJOL6gJTW5t9AwoJ8jxmwO8xgUbk+7s38VATSuaV/RiIfXfGFv34CT7AY1gRxm1og9jjP6qkFMyZiO6M+lwrJIlHKTOKxw+xc15w/tIssUkeflzAcrkkNGzT8sBL39BoQOo9RTrMD2QL weather-balloon@wesley.iplantcollaborative.org"
   - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDiAFmfG+7QP4WK6EwiKo+v34bG9DywF6f0eXym5FJkc6NQPDeG13F/kZSayFqB4IhD40nEofuawaMsPHsjJpjUAXwwgfuPUgnLjiPiFzrxdynnaevsCEzen8TXgOfvhUMgZTz9WiU7SGzSz5Sum8FncjKVjk0uVxbX6K/0FlVCm554uCEkY1l7yP59GpFDVGrDtDDUxmf+MM9862zoMWo5v8ekiFOlnILQONS0DojAoCWCb9TGmI833kKqJtVNh1ZbDdTjL/iR2o1bEbBXN+2CoPA1ZDaE25O+BOGAKY0wAL+Yc8pzeKWV7N34KK4L91XfcGPPQWggZ7O0BgHg57Ez root@lofn ansible"
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCyORkWztT4EXpwP5T3voIL/hK683RlOrGh794CSklUiFxbM/Iag6TYqDV3diNmVLfNWXvDD/WATYrsn6bijDpMn7qPBwKeuTI1f2j3tBMluUw3dsavlC2VIyyIEbL0PnLrgiXa4/OpYxZmuD9GIf9eDlb6xYsFQ3B8ZimCCX6vqUVj8gyVNEt7JMWgrx8q1D0u0jWvN2wAMoSD9epMN0IpWEMB9cwNEHbU1Jt5upm1pTC3np7o008aZxJios05iVLdCkj5bTl/ZVxJ5ShIpECBW+h5I2y9MTzeCFOAAmqsbFJN3rjI2Q6u3DjPmvh3y4aF1wGOaYnQMbpnLTqACRsS0D6IofIfuxzSCABgCLc+R4g2IqimcnhGrp62qzyGW/bLkzxEZBhKwudl4fl+ghTqDAPcCQwssvbE466CuWaBuEvfdEQ+yHTe3Fgezri4C4nhF53ZzWynUM/9ajtkzSV53fU6W4RAdYdipsXKzXDa25y5maff2AnhrvewDuVY4mt4pba13ihw2rf3p/RWmHBdN4nulZkCWdm6eQ1VgDc+shU+LE9GOTOQNzibVO1n3mAdjxJyX3RYj/egSUlYqNbBeEAAbwmbFmJTuTWQgm63IMnliPrgJjLQtuU9ZV+SJUxyoV/6qPPZA0L2DYes4iWeMW43uVFOck8Yjik/d/Ar0Q== mattd@IPC-MARA.local"
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDOQH0DvLRmg1OTfT2tCvIovV6TXtHjT4ZztpxrenmbMxWLNsVwzcni/37py0mVEDg/zpwzVi85DyTnXbwgIwr3reqjXW9D+6wnzEiRAwDUIr2fzcGs44Mgt8tnXR02MgLn0s4r73a88hgbUulTvpT0c9yi45+klqscmF63nLdPFkf19rP49IPwsoqcwIrlCarBWSEqn0oQIrBjrpWvIcV/3nxRQpynZGT049UXPm12lSjUmxsAdSuGWHk0yfjWU5uYEPxWyoJuKcQoC367DV1OEXHOquErBc38rRvql6vTOADisGrDta4jN6MTsbVB4Mw6oRbz8L9j9sx7bHa/If0f amercer@iPlants-MacBook-Pro.local"
```

Comment:
The next version of ansible should look at pulling this sort of "iPlant//Installation specific" stuff OUT of the codebase.. It'd be better served in a configuration file (Or some other solution that isn't in the atmosphere code base..)